mmv1/products/networkservices/HttpRoute.yaml

```diff
@@ -0,0 +1,507 @@
+    send_empty_value: true
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties: 
+        - !ruby/object:Api::Type::Array
+          name: matches
+          description: |
+            A list of matches define conditions used for matching the rule against incoming HTTP requests.
+          send_empty_value: true
+          item_type: !ruby/object:Api::Type::NestedObject
+            properties:
+              - !ruby/object:Api::Type::Boolean
+                name: ignoreCase
+                description: |
+                  Specifies if prefixMatch and fullPathMatch matches are case sensitive.
+                send_empty_value: true
+              - !ruby/object:Api::Type::String
+                name: fullPathMatch
+                description: |
+                  The HTTP request path value should exactly match this value.
+                exactly_one_of:
+                  - fullPathMatch
+                  - prefixMatch
+                  - regexMatch
+              - !ruby/object:Api::Type::String
+                name: prefixMatch
+                description: |
+                  The HTTP request path value must begin with specified prefixMatch.
+                exactly_one_of:
+                  - fullPathMatch
+                  - prefixMatch
+                  - regexMatch
+              - !ruby/object:Api::Type::String
+                name: regexMatch
+                description: |
+                  The HTTP request path value must satisfy the regular expression specified by regexMatch after removing any query parameters and anchor supplied with the original URL.
+                exactly_one_of:
+                  - fullPathMatch
+                  - prefixMatch
+                  - regexMatch   
+              - !ruby/object:Api::Type::Array
+                name: queryParameters
+                description: |
+                  Specifies a list of query parameters to match against. 
+                send_empty_value: true
+                item_type: !ruby/object:Api::Type::NestedObject
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: queryParameter
+                      description: |
+                        The name of the query parameter to match.
+                      send_empty_value: true
+                    - !ruby/object:Api::Type::String
+                      name: exactMatch
+                      description: |
+                        The value of the query parameter must exactly match the contents of exactMatch.
+                      exactly_one_of:
+                        - exactMatch
+                        - regexMatch
+                        - presentMatch
+                    - !ruby/object:Api::Type::String
+                      name: regexMatch
+                      description: |
+                        The value of the query parameter must match the regular expression specified by regexMatch.
```

Comment:
Can we add the following to the description?

`For regular expression grammar, please see https://github.com/google/re2/wiki/Syntax`